CesiumGeospatial/include/CesiumGeospatial/GlobeAnchor.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "Library.h"
+
+#include <CesiumGeospatial/Ellipsoid.h>
+
+#include <glm/mat4x4.hpp>
+
+#include <optional>
+
+namespace CesiumGeospatial {
+
+class LocalHorizontalCoordinateSystem;
+
+/**
+ * @brief Anchors an object to the globe by definining a transformation from the
```

Comment:
```suggestion
 * @brief Anchors an object to the globe by defining a transformation from the
```

I'll push a fix for this typo.